specification/consumption/resource-manager/Microsoft.Consumption/2017-11-30/consumption.json

```diff
@@ -0,0 +1,469 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-11-30",
+    "title": "ConsumptionManagementClient",
```

Comment:
This should really just be `"title": "Consumption Management",`

Reply:
Sure changed now